src/Duo.Clients.Wpf/Messaging/Handlers/ApriManutenzioneProdottoMessageHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+using Duo.Clients.Wpf.Presentation;
+using System.Windows;
+using Topics.Radical.ComponentModel.Messaging;
+using Topics.Radical.Messaging;
+using Topics.Radical.Windows.Presentation.ComponentModel;
+
+namespace Duo.Clients.Wpf.Messaging.Handlers
+{
+    class ApriManutenzioneProdottoMessageHandler : AbstractMessageHandler<ApriManutenzioneProdottoMessage>, INeedSafeSubscription
+    {
+        readonly IViewResolver viewResolver;
+        readonly IConventionsHandler conventions;
+
+        public ApriManutenzioneProdottoMessageHandler(IViewResolver viewResolver, IConventionsHandler conventions)
+        {
+            this.viewResolver = viewResolver;
+            this.conventions = conventions;
+        }
+        public override void Handle(object sender, ApriManutenzioneProdottoMessage message)
+        {
+            var view = viewResolver.GetView<ManutenzioneProdottoView, ManutenzioneProdottoViewModel>(vm =>
+                                                                                                        {
+                                                                                                            vm.Id = message.Id;
+                                                                                                            vm.Version = message.Version;
+                                                                                                            vm.Codice = message.Codice;
+                                                                                                            vm.Descrizione = message.Descrizione;
+                                                                                                            vm.Spessore = message.Spessore;
+                                                                                                        });
```

Comment:
In questo caso invece che passare i dati da modificare con il messaggio io passerei il solo Id e farei si che il VM recuperi dal server la View ad hoc che gli serve per visualizzare la cosa da modificare. In questo caso stai partendo dal presupposto che la view usata per visualizzare il dato contenga tutto quello che serve per fare anche le modifiche. Magari in questo caso va benissimo, facciamoci un ragionamento.